extensions/gitpod-shared/src/analytics.ts

```diff
@@ -38,8 +38,9 @@ export type GitpodAnalyticsEvent =
 		action: 'share' | 'stop-sharing' | 'stop' | 'snapshot' | 'extend-timeout';
 	}> |
 	GAET<'vscode_execute_command_gitpod_ports', {
-		action: 'private' | 'public' | 'preview' | 'openBrowser';
+		action: 'private' | 'public' | 'preview' | 'openBrowser' | 'urlCopy';
 		isWebview?: boolean;
+		userOverride?: string; // 'true' | 'false'
```

Comment:
(nit) `userOverride?: 'true' | 'false'` - ?

Reply:
`String(someBoolValue)` is not allowed if we defined `userOverride` with `'true' | 'false'` in typescript